src/pages/AdminHomePage.tsx

```diff
@@ -161,9 +165,23 @@ const AdminHomePage = () => {
         )
     }
 
+    const renderProgressSpinner = () => {
+        return (
```

Comment:
Clean this code a bit (braces on lines 169 and 177 are not needed, also indent correctly). If you do not have it install prettier extension for vsc, it automates things like this.